r4e-image.sh

```diff
@@ -74,29 +78,33 @@ composer-cli blueprints push $BLUEPRINT_FILE
 # create image
 echo "creating image $IMAGE_NAME"
 composer-cli compose start $IMAGE_NAME rhel-edge-commit > temp.out
-BUILD_ID=`cat temp.out | awk '{print $2}'`
+BUILD_ID=$(awk '{print $2}' temp.out)
 echo "waiting for build $BUILD_ID to be ready..."
-while [ "`composer-cli compose status | grep $BUILD_ID | grep -c RUNNING`" != "0" ] ; do sleep 5 ; done
-if [ "`composer-cli compose status | grep $BUILD_ID | grep -c FINISHED`" == "0" ] ; then
+while [ "$(composer-cli compose status | grep $BUILD_ID | grep -c RUNNING)" != "0" ] ; do sleep 5 ; done
+if [ "$(composer-cli compose status | grep $BUILD_ID | grep -c FINISHED)" == "0" ] ; then
     echo "image composition failed"
     echo "check 'composer-cli compose status'"
     exit 1
 fi
 
 # extract image to web server folder
-echo "saving image to web folder"
+echo "saving image to web folder $IMAGE_FOLDER"
 composer-cli compose image $BUILD_ID
 sudo mkdir $IMAGE_FOLDER
 sudo tar -xvf $BUILD_ID-commit.tar -C $IMAGE_FOLDER
 
 # create kickstart file and copy to web server
+if [ "$ARCH" = "aarc64" ]; then
```

Comment:
where do we get ARCH from?

Reply:
it is set in the calling script (k4e-agent-sti.sh) and it is obtained from the running OS itself. there is an assumption here that we'll use rhel-8.5 x86_64 to create image for x86_64, and we need to have aarch64 for creating aarch64 image.

I'll define it explicitly here not to rely on the caller to pass it.